src/common/assert.cpp

```
@@ -35,17 +35,18 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #include <execinfo.h>
 #endif
 
+#include <array>
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)